sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -75,13 +86,31 @@ describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
       // Student created with expected first name, last name and more importantly the updated date
       // to fall between the most recent bridge file date and the mocked current date.
       const student = await createStudentWithExpectedData(simsDataUpdatedDate);
+
       // Student has submitted an application.
-      await saveFakeApplicationDisbursements(
+      const application = await saveFakeApplicationDisbursements(
         db.dataSource,
         { student },
-        { offeringIntensity: OfferingIntensity.partTime },
+        {
+          offeringIntensity: OfferingIntensity.partTime,
+          updatedAt: simsDataUpdatedDate,
+        },
       );
 
+      // Student has a restriction.
+      const provincialRestriction = await db.restriction.findOne({
+        select: { id: true, restrictionCode: true },
+        where: {
+          restrictionType: RestrictionType.Provincial,
+        },
+      });
```

Comment:
I would say this query can be executed in before all so that it could potentially be used in other tests if required.